framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -986,7 +1039,8 @@ private void handleExpr(AnnotationBuilder builder, String name,
                     builder.setValue(name, arr);
                 }
             } else {
-                ErrorReporter.errorAbort("StubParser: unhandled annotation attribute type: " + expr + " and expected: " + expected);
+                ErrorReporter.errorAbort("StubParser: unhandled annotation attribute type: " + expr +
+                        " and expected: " + expected);
```

Comment:
The Checker Framework currently uses the Sun style guide, as does javac on which the CF builds.
This pull request is consistent with the overall style of the rest of the code.
If you don't want to merge the PR just let me know. I don't see a point in discussing the coding style here and don't see a value in partially changing the coding style.